notebook.ipynb

```diff
@@ -0,0 +1,174 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Clox"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----------------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Environment Check"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Checking that the notebook is running on Google Colab or not."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "try:\n",
+    "  import google.colab\n",
+    "  !{sys.executable} -m pip -q -q install clox\n",
+    "except:\n",
+    "  pass"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Clock"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from clox.functions import run_clock"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "clock_parameters = {\n",
+    "    \"timezone\": None,\n",
+    "    \"country\": None,\n",
+    "    \"v_shift\": 0,\n",
+    "    \"h_shift\": 0,\n",
+    "    \"face\": 1,\n",
+    "    \"no_blink\": False,\n",
+    "    \"vertical\": False,\n",
+    "    \"hide_date\": False,\n",
+    "    \"hide_timezone\": False,\n",
+    "    \"am_pm\": False,\n",
+    "    \"date_system\": \"GREGORIAN\",\n",
+    "    \"date_format\": \"FULL\",\n",
+    "    \"offset_local\": 0,\n",
+    "    \"offset_timezone\": 0\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run_clock(**clock_parameters)"
+   ]
+  },
```

Comment:
I suggest putting the parameter directly into the function as variables.
I know there are people who would be confused if we use `**kwargs` but will be ok with direct use of parameters.

Reply:
I understand your concern. However, given the function's numerous parameters and the potential need to handle threads for async mode (as you mentioned), I believe it's best to define the parameters separately.